playbooks/openstack-flavor-setup.yml

```diff
@@ -18,9 +18,9 @@
   environment: "{{ deployment_environment_variables | default({}) }}"
   user: root
   tasks:
-    - name: Install shade
+    - name: Install OpenStack SDK
```

Comment:
This task is unnecessary - shade/openstacksdk are installed into the runtime by OSA:

master: https://github.com/openstack/openstack-ansible/blob/master/requirements.txt#L26
rocky: https://github.com/openstack/openstack-ansible/blob/stable/rocky/requirements.txt#L26
queens: https://github.com/openstack/openstack-ansible/blob/stable/queens/requirements.txt#L12

Only pike requires this task.

Reply:
done